EIPS/eip-5344.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 5344
+title: Soulmates Token
+description: Representing soulmates by collectible soulbound tokens.
+author: Jun Kawasaki (@jun784)
+discussions-to: https://ethereum-magicians.org/t/10132
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-27
+requires: 721, 5114
+---
+
+## Abstract
+
+A soulmates token is a countable soulbound (non-transferrable) non-fungible token that has a fixed list of addresses that collectively own the token.
+
+## Specification
+```solidity
+interface IERC1313 is IERC5114 {
+  // returns a current number of mates.
+  // this function **MUST** return the latest number of tokens.
+  function current(uint256 counter) external view returns (uint256)
+  
+  // increment number of mates.
+  // this function **MUST** increment 
+  // this function **MUST** return the incremented number.
+  function increment(uint256 counter) external view returns (uint256)
+  
+  // burn soulbound token.
+  // this function **MUST** only be able to operate by the token holder.
+  function burn(uint256 tokenId) external virtual return 
```

Comment:
This function is incompatible with EIP-5114 which states that `ownerOf(tokenId)` cannot change once it is minted.